File: python-puka/send.py
#!/usr/bin/env python
import puka

client = puka.Client("amqp://localhost/")
promise = client.connect()
client.wait(promise)


promise = client.queue_declare(queue='hello')
client.wait(promise)

promise = client.basic_publish(exchange='',
                               routing_key='hello',
                               body="Hello World!")
client.wait(promise)

print " [x] Sent 'Hello World!'"
client.close()

File: javascript-nodejs/src/send.js
#!/usr/bin/env node

var amqp = require('amqplib/callback_api');

amqp.connect('amqp://localhost', function(error, connection) {
  if (error) throw error;
  connection.createChannel(function(err, channel) {
    if (err) throw err;
    var queue = 'hello';
    var msg = 'Hello World!';

    channel.assertQueue(queue, {durable: false});
    channel.sendToQueue(queue, Buffer.from(msg));
    console.log(" [x] Sent %s", msg);
  });
  setTimeout(function() { connection.close(); process.exit(0) }, 500);
});
